tests/docker/gitlab.Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.9-slim
```

Comment:
do we need to include gitlab specific files in github as sync would be one way (from gh to gitlab) assuming code will still be committed in gh but a hook on merge will sync it to gitlab triggering gitlab pipeline.

Reply:
Discussed offline, synopsis is that the GitHub repo is still the Source Of Truth, I’m using the gitlab “includes” trick so that we don’t have to keep the gitlab pipeline code here.